docs/versioned_docs/version-zenith/developer-guide/python/753720-arguments-return-values.mdx

```diff
@@ -7,10 +7,12 @@ title: "Functions"
 
 import PartialExperimentalDocs from '../../partials/_experimental.mdx';
 
-# Functions
+# Functions, Arguments and Return Values
```

Comment:
I changed this to just "Functions" on purpose. Arguments and return values **are a part** of any function in any language, including in dagger. The longer title is unnecessary in my opinion.

Reply:
Good point, changed in all